app/controllers/catalog_controller.rb

```diff
@@ -58,25 +58,19 @@ def x_button
   end
 
   def servicetemplate_edit
-    assert_privileges(params[:pressed]) if params[:pressed]
-    checked = find_checked_items
+    assert_privileges(params[:pressed]) if params[:pressed].present?
+    checked_id = find_checked_id_items_with_rbac(ServiceTemplate).first || test_item_with_rbac(ServiceTemplate, params[:id])
     @sb[:cached_waypoint_ids] = MiqAeClass.waypoint_ids_for_state_machines
-    checked[0] = params[:id] if checked.blank? && params[:id]
-    @record = checked[0] ? find_by_id_filtered(ServiceTemplate, checked[0]) : ServiceTemplate.new
+    @record = checked_id || ServiceTemplate.new
```

Comment:
` -    @record = checked[0] ? find_by_id_filtered(ServiceTemplate, checked[0]) : ServiceTemplate.new `
this was right